docs/contributor/03-10-hyperscaler-account-pool.md

```diff
@@ -82,3 +84,34 @@ metadata:
     tenant-name: {TENANT_NAME}
     hyperscaler-type: "gcp_cf-sa30"
 ```
+
+## Secret Bindings Selection Rules
+
+<!-- rules Overview -->
+HAP evaluates a set of rules to determine what labels to use when querying secret bindings. Input to the rules consists of an SKR's plan, hyperscaler and region. If evaluated to true the rules modify or add labels used in the secret bindings resource query. There are four possible rules to configure:
+* `hap.platformRegionRule` - if evaluated to true the `_<PLATFOR_REGION>` is appended to the `hyperscaler-type` label when searching, refered to as platform region based search,
```

Comment:
Are `region` rules mutually exclusive?

Reply:
We agreed that all the rules can be inclusive because they are independent of each other.

Reply:
So we need to put it here explicitly and possibly add an example,

Reply:
Done. Please see the newest changes.